src/app/components/applications/SSV/RegisterValidatorHome/components/OfflineKeyShareGeneration/OfflineKeyShareGeneration.tsx

```diff
@@ -82,14 +108,41 @@ const OfflineKeyShareGeneration = () => {
         setSelectedBox(id);
     };
 
+    const changeWithdrawalAddressHandler = (e: any) => {
+        const { value } = e.target;
+        setWithdrawalAddress(value);
+        validateAddressInput(value, setAddressValidationError);
+        setTextCopied(false);
+        setConfirmedWithdrawalAddress(false);
+    };
+
+    const showCopyButtonCondition = selectedBox === 1 || (selectedBox === 3 && withdrawalAddress && !addressValidationError.shouldDisplay && confirmedWithdrawalAddress);
+    const commandCli = selectedBox === 1 ? cliCommand : dkgCliCommand;
+    const buttonLabelCondition = selectedBox === 1 || selectedBox === 3 && operatorsAcceptDkg || selectedBox === 0;
+    const cliCommandPanelCondition = selectedBox === 1 || selectedBox === 3 && operatorsAcceptDkg;
+    const buttonLabel = buttonLabelCondition ? 'Next' : 'Change Operators';
+    const submitFunctionCondition = selectedBox === 3 && !operatorsAcceptDkg;
+
+    const disabledCondition = () => {
+        if (selectedBox === 1 || selectedBox === 3 && operatorsAcceptDkg) {
+            return !textCopied;
+        } else if (selectedBox === 0) {
+            return true;
+        } else {
+            return false;
+        }
+    };
```

Comment:
Can disabled condition be a state which changes depending on selected box and operatorsAcceptDkg?